src/index.tsx

```diff
@@ -178,6 +178,17 @@ function App() {
   const onMapClickHandler = useCallback((info: PickingInfo) => {
     // We added this flag to prevent the hover event from firing after a
     // click event.
+    if (typeof info.coordinate !== "undefined") {
+      let x = info.coordinate[0];
+      while (x < 180) {
+        x += 360;
+      }
+      while (x > 180) {
+        x -= 360;
+      }
+      info.coordinate[0] = x;
```

Comment:
Do you know how often this happens? This kind of coordinate shuffling won't work if we support alternate coordinate systems

Reply:
The only time this happens is when the user is zoomed out and can see the whole earth wrapped around, and then when they click on the far right side of the map the x coordinate can exceed 180 (and likewise for the left side, you can end up less than -180).  if you've got plans to support other coordinate systems, we should certainly exclude this and leave that sort of logic for the to the person writing the callback to implement.

Reply:
Yeah I think for now let's remove this coordinate shuffling but document that in some cases the values can be outside the expected range.

Reply:
I've added a note to the docstring of on_click function indicating the X/Longitude may be less than or greater than expected, with an example of what will happen if the map is wrapped around and the user clicks on the left or the right part of the earth (hopefully it makes sense, I'm very open to any re-wording/clarification)